fw/preamp/src/channel.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * AmpliPi Home Audio
+ * Copyright (C) 2021 MicroNova LLC
+ *
```

Comment:
We need to give a little info about each of these modules, first stab:
Configure each of the preamp controller's 6 channels. 

Each channel has several different configurable properties:
- Attenuation
- Standby
- Mute
- Audio Source